epick_controllers/src/epick_gripper_action_controller.cpp

```diff
@@ -36,6 +36,21 @@
 #include <hardware_interface/loaned_command_interface.hpp>
 #include <hardware_interface/loaned_state_interface.hpp>
 
+namespace
```

Comment:
Is it necessary to put this in the default namespace or this a required code style thing?

Reply:
More precisely, this function is within an anonymous namespace, which makes it so the function is only visible within this particular source file. We do this to control what parts of the library's API are exposed to users who might link against the targets in these packages.